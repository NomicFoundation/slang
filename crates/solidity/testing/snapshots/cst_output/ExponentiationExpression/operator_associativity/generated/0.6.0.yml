# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Source: >
  1  │ x ** y ** z                                                                      │ 0..11

Errors: []

Tree:
  - ExponentiationExpression (Rule): # 0..11
      - Identifier (Token): "x" # 0..1
      - StarStar (Token): "**" # 2..4
      - ExponentiationExpression (Rule): # 5..11
          - Identifier (Token): "y" # 5..6
          - StarStar (Token): "**" # 7..9
          - Identifier (Token): "z" # 10..11
