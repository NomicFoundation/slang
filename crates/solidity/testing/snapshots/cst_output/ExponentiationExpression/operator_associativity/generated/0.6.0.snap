# This file is generated automatically by infrastructure scripts. Please don't edit by hand.

Errors: 0

█ Rule(ExponentiationExpression) - (0..11)        │
├──► Token(Identifier) - (0..1)                   │ x
├──► Token(StarStar) - (2..4)                     │ **
└──█ Rule(ExponentiationExpression) - (5..11)     │
   ├──► Token(Identifier) - (5..6)                │ y
   ├──► Token(StarStar) - (7..9)                  │ **
   └──► Token(Identifier) - (10..11)              │ z
